web/pgadmin/static/js/SchemaView/FormViewTab.jsx

```diff
@@ -0,0 +1,45 @@
+/////////////////////////////////////////////////////////////
+//
+// pgAdmin 4 - PostgreSQL Tools
+//
+// Copyright (C) 2013 - 2025, The pgAdmin Development Team
+// This software is released under the PostgreSQL Licence
+//
+//////////////////////////////////////////////////////////////
+
+import { Tab } from '@mui/material';
+import React, { useContext, useState } from 'react';
+import { useFieldOptions, useSchemaStateSubscriber } from './hooks';
+import { SchemaStateContext } from './SchemaState';
+import PropTypes from 'prop-types';
+
+export default function FormViewTab({tabGroup, idx, tabValue, ...props}) {
+  const accessPath = [tabGroup.id];
+  const [, setRefreshKey] = useState(0);
```

Comment:
Code Smell: The return value of "useState" should be destructured and named symmetrically